libretroshare/src/services/p3gxsforums.cc

```diff
@@ -192,221 +192,202 @@ RsSerialiser* p3GxsForums::setupSerialiser()
 
 void p3GxsForums::notifyChanges(std::vector<RsGxsNotify *> &changes)
 {
-#ifdef GXSFORUMS_DEBUG
-	std::cerr << "p3GxsForums::notifyChanges() : " << changes.size() << "changes to notify" << std::endl;
-#endif
+	RS_DBG2(changes.size(), " changes to notify");
 
 	std::vector<RsGxsNotify *>::iterator it;
 	for(it = changes.begin(); it != changes.end(); ++it)
 	{
-		RsGxsMsgChange *msgChange = dynamic_cast<RsGxsMsgChange *>(*it);
-
-		if (msgChange)
+		rs_view_ptr<RsGxsNotify> gxsChange = *it;
```

Comment:
I understand that you'd like to use this across fonction calls, but here it's just a local variable. It looks a bit over complicated for what it actually brings.

Reply:
Ok, I can change to an unnannotated pointer if you prefer